src-test/org/etools/j1939_84/controllers/part01/Part01Step15ControllerTest.java

```diff
@@ -302,44 +321,24 @@ public void testGetStepNumber() {
     }
 
     @Test
+    @TestDoc(value = @TestItem(verifies = "6.1.15.1", description = "Verifies that there is a single 6.1.15 step"))
     public void testGetTotalSteps() {
         assertEquals("Total Steps", 0, instance.getTotalSteps());
```

Comment:
"Verifies that there is a single 6.1.15 step", but then is verifies that there are zero steps.

Reply:
Removed - should not have been added.  Number of steps represents the times the progress bar will be updated.  They don't care.